Python/Product/PythonTools/visualstudio_py_debugger.py

```diff
@@ -1459,7 +1459,7 @@ def get_frame_list(self):
                 # collect globals used locally, skipping undefined found in builtins
                 f_globals = cur_frame.f_globals
                 if f_globals: # ensure globals to work with (IPy may have None for cur_frame.f_globals for frames within stdlib)
-                    self.collect_variables(vars, f_globals, cur_frame.f_code.co_names, treated, skip_unknown = True)
+                    self.collect_variables(vars, f_globals, cur_frame.f_code.co_name, treated, skip_unknown = True)
```

Comment:
This is actually a significant change - `co_name` is the name of the code object, while `co_names` is a tuple containing globals referenced by the code object. If Jython does not support `co_names`, this will need to become a `getattr(cur_frame.f_code, 'co_names', ())` (ideally on a separate line, to keep this one from getting too long)